src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -304,7 +307,7 @@ export const MousePopulationsModel = types
     function addSettingsAnnotation(label: string, labelXOffset: number) {
       const now = interactive ? interactive.environment.date : 0;
       const timeSinceLastAnnotation = now - lastSettingsAnnotationDate;
-      const labelYOffset = timeSinceLastAnnotation < 30 ? 55 : 30;
+      const labelYOffset = timeSinceLastAnnotation < 30 ? 170 : 145;
```

Comment:
We might want to add constants for these hard-coded values.  I'll certainly never remember what they mean :)